module3-crud-and-data-models/r1-schema-and-data-models/README.md

```diff
@@ -0,0 +1,124 @@
+## Introduction To Database Schemas
+
+A database schema is a blueprint or architecture of how our data will look. It doesn’t hold data itself, but instead describes the shape of the data and how it might relate to other tables or models.
+
+For example, this is a `customer` schema in MongoDB using Mongoose.
+
+```js
+const mongoose = require("mongoose");
+
+const customer = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  zipcode: {
+    type: Number,
+  },
+});
+
+module.exports = mongoose.model("Customer", customer);
+```
+
+The same `customer` schema in a MySql database.
+
+```sql
+CREATE TABLE customer (
+ id INT AUTO_INCREMENT PRIMARY KEY,
+ name VARCHAR(50) NOT NULL,
+ zipcode INT(15) default NULL,
+)
+
+```
```

Comment:
This is the SQL command to create a database.
You can also give an example to show a schema in PostgreSQL using Sequelize, similar to how you've shown MongoDB schema using Mongoose. So that students are not confused between schema using an ORM and SQL commands.